dom.bs

```diff
@@ -1488,7 +1488,7 @@ can be used to explore this matter in more detail.
 
 <h3 id=node-trees>Node tree</h3>
 
-<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{ShadowRoot}}, {{Element}}, {{Text}},
```

Comment:
Mind splitting the ShadowRoot changes into a separate commit for landing? Same PR is fine, I'll merge the sum of it now.